Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/security/LoginController.java

```diff
@@ -13,14 +17,19 @@
 public class LoginController {
   /** Riceve le chiamate all'endpoint REST per l'autenticazione. */
   @GetMapping
-  public Map<String, String> login(HttpSession session, Authentication authentication) {
+  public Map<String, Object> login(HttpSession session, Authentication authentication) throws BusinessException {
     if(authentication != null) {
       var authorities = authentication.getAuthorities();
       var adminAuthority = ProdulyticsGrantedAuthority.ADMIN.getAuthority();
       var admin = authorities.stream().anyMatch(auth -> auth.getAuthority().equals(adminAuthority));
-      return Map.of("accessToken", session.getId(), "admin", Boolean.toString(admin));
+      var username = authentication.getName();
+      return Map.of(
+          "accessToken", session.getId(),
+          "admin", admin,
+          "username", username
```

Comment:
Questi dovrebbero probabilmente essere in una query a parte.

Reply:
dici perché intanto uno potrebbe aver disattivato l'account?

Reply:
No, perchè logicamente ottenere i dati sulla sessione attuale mi sembra diverso dall'effettuare il login.